bridge_adaptivity/bridge_lti/models.py

```diff
@@ -32,19 +31,19 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
-class LtiConsumer(ModelFieldIsDefaultMixin, models.Model):
+class LtiConsumer(models.Model):
     """
     Model to manage LTI source providers.
 
     Content source connections.
     """
 
-    IS_DEFAULT_FIELD = 'is_active'
     name = fields.CharField(max_length=255, blank=True, null=True, unique=True)
     provider_key = models.CharField(max_length=255)
     provider_secret = models.CharField(max_length=255)
     lti_metadata = fields.CharField(max_length=255, null=True, blank=True)
     host_url = models.URLField(max_length=255, null=True)
+    o_auth_client = models.ForeignKey('api.OAuthClient', null=True)
```

Comment:
I would set this field to be mandatory and not empty to avoid creating Consumer without oauth_client.

Reply:
it was not possible, because migration will not work. `null=True` is ok, because form will require this field because it has `blank=False`.

Reply:
Make sense, thanks for the explanation.